proto/plugin/plugin.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package helperPlugin;
+option go_package = "../../pkg/helperPlugin";
+
+message Empty {}
+
+message ConfigsRequest {
+    map<string,string> configs = 1;
+}
+
+service SpiffeHelper {
+    rpc PostConfigs(ConfigsRequest) returns (Empty) {};
```

Comment:
what are those configs? only paths? 
I feels like this plugin ends is to notify "something" about there are updated on bundles, 
so we must know what is updated, and what to do based on each plugin config,
So the Service may be something like `Notifier`, and your PostConfig, may be `Notify`
what do you think?

Reply:
Yes, most of the configs are paths to certs. The current approach is to execute the plugins on each cert update (and let them do what they want with those certs), but yes, it is possible to keep them running and send a notification on updates.